user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/API_Reference/GQIPage.md

```diff
@@ -16,4 +16,7 @@ Represents a single page of rows. Every query result consists of one or more suc
 | Property | Type | Required | Description |
 | -------- | ---- | -------- | ----------- |
 | Rows | [GQIRow](xref:GQI_GQIRow)[] | Yes | Rows of the page. |
-| HasNextPage | boolean  | No | `True` if GQI can fetch additional pages, `False` if the current page is the last page. |
+| HasNextPage | boolean  | No | `true` if there are more pages that GQI can fetch, otherwise 'false' (default). |
+
+> [!TIP]
+> Learn how `HasNextPage` can be used to apply paged data retrieval through following example: [paged data retrieval example](xref:GQI_IGQIDataSource#paged-data-retrieval)
```

Comment:
Might be good to mention here briefly why you would care.
The link to the example can be integrated in the text.

```suggestion
> When dealing with large amounts of data, it is recommended to spread the data across multiple pages. See [this example](xref:GQI_IGQIDataSource#paged-data-retrieval) on how the `HasNextPage` property can be used to enable paged data retrieval.
```